### PR TITLE
Make `registerTech` add that tech to the default `techOrder`

### PR DESCRIPTION
**Pull request by BrandonOCasey**
## Description
Make `registerTech` add that tech to the default `techOrder`

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [x] If necessary, more likely in a feature request than a bug fix
  - [x] Change has been verified in an actual browser (Chome, Firefox, IE)
 - [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: src/js/tech/tech.js [hunk 1]
```diff
@@ -802,6 +802,9 @@ class Tech extends Component {
     middleware.use('*', {name, tech});
 
     Tech.techs_[name] = tech;
+    if (name !== 'Tech') {
+      Tech.defaultTechs_.push(name.toLowerCase());
```
Comment (gkatsev) [inline src/js/tech/tech.js]: techOrder expects them to be in camel case.

**Comment by BrandonOCasey** [comment]
@gkatsev I changed it to convert to `camelCase` but I am wondering if we can't just add it to TechOrder as is. Seems like we call `toTitleCase` on `techOrder` when we use it anyway. Maybe we should even make the `techOrder` search case insensitive? That last idea seems like it might be a bit out of scope here though.

**Comment by gkatsev** [comment]
We should just do what we did for components where it doesn't care whether it's camel or title cased.

**Comment by BrandonOCasey** [comment]
Do we want to do that in this PR? if not I will make another to do it so we can just get this one merged to get `videojs-flash` RC.0 out

**Comment by gkatsev** [comment]
I guess doing it in another PR would be cleaner

**Comment by BrandonOCasey** [comment]
#3986

**Pull request merged**